### PR TITLE
Request pip via trusted click

### PR DESCRIPTION
**Pull request by lukebjerring**
Fixes a bug in the picture-in-picture IDL test, which intends to obtain a `PictureInPictureWindow` object for testing IDL.

### CODE REVIEW

**Review by beaufortfrancois — APPROVED** [review]

File: picture-in-picture/idlharness.window.js [hunk 1]
```diff
@@ -11,7 +12,8 @@ promise_test(async () => {
     const video = await loadVideo();
     document.body.appendChild(video);
     self.video = video;
-    self.pipw = await video.requestPictureInPicture();
+    video.autoplay = true;
```
Comment (beaufortfrancois) [inline picture-in-picture/idlharness.window.js]: Nit: `video.autoplay` shoud not be needed.

File: picture-in-picture/idlharness.window.js [hunk 2]
```diff
@@ -11,7 +12,8 @@ promise_test(async () => {
     const video = await loadVideo();
     document.body.appendChild(video);
```
Comment (beaufortfrancois) [inline picture-in-picture/idlharness.window.js]: Nit: This should not be needed as well.

File: picture-in-picture/idlharness.window.js [hunk 3]
```diff
@@ -11,7 +12,8 @@ promise_test(async () => {
     const video = await loadVideo();
```
Comment (beaufortfrancois) [inline picture-in-picture/idlharness.window.js]: It could be as simple as `self.video = await loadVideo();`

**Comment by lukebjerring** [comment]
@kereliuk - this actually fails, since `click` doesn't seem to be implemented. Where are we at with simulating user clicks via `testdriver.js` ?

EDIT: Calling `click` fails when running the test using `./wpt serve` and loading the URL locally:

    Uncaught (in promise) Error: unimplemented
        at Object.click (testdriver.js:185)
        at Object.click (testdriver.js:119)
        at requestAnimationFrame (picture-in-picture-helpers.js:14)

It fails differently using `./wpt run chrome picture-in-picture/idlharness.window.html`:

    Uncaught TypeError: Cannot read property 'postMessage' of null
        at Object.window.test_driver_internal.click (testdriver.js:269)
        at Object.click (testdriver.js:119)
        at requestAnimationFrame (picture-in-picture-helpers.js:14)

**Comment by kereliuk** [comment]
@lukebjerring testdriver cannot be used with wpt serve, only with wptrunner

**Comment by lukebjerring** [comment]
OK so before I get too tetchy about it not being documented, there's already an issue open for that:
https://github.com/web-platform-tests/wpt/issues/10330

**Comment by lukebjerring** [comment]
OK so if I run this locally with `wpt run`, in its current state, testdriver.js behaves as expected (the click event happens).

**Pull request merged**